test/e2e/resources/certificate_authority_activation.yaml

```diff
@@ -10,4 +10,5 @@ spec:
   certificate:
     namespace: $CERTIFICATE_SECRET_NAMESPACE
     name: $CERTIFICATE_SECRET_NAME
-    key: $CERTIFICATE_SECRET_KEY
\ No newline at end of file
+    key: $CERTIFICATE_SECRET_KEY
+  status: ACTIVE
```

Comment:
This field shouldn't be submitted by the e2e test process, it should be populated by the controller it self

Reply:
Ohh initially i thought this was the top level status field